changelog_entry.yaml

```
@@ -0,0 +1,6 @@
+- bump: patch
+  changes:
+    removed:
+    - .vscode folder
```

and below, and newline at the bottom may fix the CI
```suggestion
      - .vscode folder
```